specification/mysql/resource-manager/Microsoft.DBforMySQL/Backups/preview/2022-09-30-preview/BackupAndExport.json

```diff
@@ -0,0 +1,373 @@
+          "type": "string"
+        }
+      },
+      "discriminator": "objectType"
+    },
+    "FullBackupStoreDetails": {
+      "description": "FullBackupStoreDetails is used for scenarios where backup data is streamed/copied over to a storage destination.",
+      "type": "object",
+      "required": [
+        "sasUriList"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupStoreDetails"
+        }
+      ],
+      "properties": {
+        "sasUriList": {
+          "description": "SASUriList of storage containers where backup data is to be streamed/copied.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "BackupSettings": {
+      "description": "Backup Settings",
+      "type": "object",
+      "required": [
+        "backupName"
+      ],
+      "properties": {
+        "backupName": {
+          "$ref": "../../../common-types/v1/common-types.json#/definitions/BackupName"
+        },
+        "backupFormat": {
+          "description": "Backup Format for the current backup. (CollatedFormat is INTERNAL – DO NOT USE)",
+          "enum": [
+            "None",
+            "CollatedFormat"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "BackupFormat",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ValidateBackupResponseProperties": {
+      "description": "ValidateBackup Response Properties",
+      "type": "object",
+      "properties": {
+        "numberOfContainers": {
+          "format": "int32",
+          "description": "Estimated no of storage containers required for resource data to be backed up.",
+          "type": "integer"
+        }
+      }
+    },
+    "BackupAndExportResponseProperties": {
+      "description": "BackupAndExport Response Properties",
+      "type": "object",
+      "properties": {
+        "datasourceSizeInBytes": {
+          "format": "int64",
+          "description": "Size of datasource in bytes",
+          "type": "integer"
+        },
+        "dataTransferredInBytes": {
+          "format": "int64",
+          "description": "Data transferred in bytes",
+          "type": "integer"
+        },
+        "backupMetadata": {
+          "description": "Metadata related to backup to be stored for restoring resource in key-value pairs.",
+          "type": "string"
+        }
+      }
+    },
+    "BackupAndExportResponse": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "The operation status",
+          "type": "string",
+          "enum": [
+            "Pending",
+            "InProgress",
+            "Succeeded",
+            "Failed",
+            "CancelInProgress",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "OperationStatus",
+            "modelAsString": false,
```

Comment:
>false [](http://example.com/codeflow?start=29&length=5)

Why marking as false? May potential break client if changes are introduced in future